### PR TITLE
feat(postgresql): add support for multiple insert in one query using createAll() of connector

### LINKED ISSUE #523: Support bulk insert in one query using newly added createAll() in connector
## Suggestion

With the addition of efficient bulk insert in one DB query feature in loopback-connector after release of [this PR](https://github.com/loopbackio/loopback-connector/pull/228), its important that we implement this method in postgres connector too as postgresql already supports `insert into ... values ().. n` dialect for multiple inserts in 1 single query.

Also relates to https://github.com/loopbackio/loopback-next/issues/3357

## Use Cases

Db query optimization. Faster insert. Less DB connection utilizations.

## Examples

No changes needed in consuming apps if they are using existing createAll method of repository

## Acceptance criteria

TBD - will be filled by the team.

### PR DESCRIPTION
**Pull request by samarpanB**
BREAKING CHANGE: This drops explicit, documented support for Node.js v10, v11, v12, v13, v15, v17

Signed-off-by: Samarpan Bhattacharya <this.is.samy@gmail.com>

1. Added support for multiple insert in one query using createAll() of connector. 
2. Updated some existing tests for name field of id column and added some too. This was done to test if the new getInsertedIds method works in those cases or not.

Fixes #523
See also https://github.com/loopbackio/loopback-next/issues/3357

## Checklist

- [x] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [x] `npm test` passes on your machine
- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](https://loopback.io/doc/en/contrib/style-guide-es6.html)
- [x] Commit messages are following our [guidelines](https://loopback.io/doc/en/contrib/git-commit-messages.html)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
     "juggler-v3": "file:./deps/juggler-v3",
     "juggler-v4": "file:./deps/juggler-v4",
     "lodash": "^4.17.4",
-    "loopback-datasource-juggler": "^3.0.0 || ^4.0.0",
+    "loopback-datasource-juggler": "^4.28.0",
```
[the remark this answers is not in the document]
Comment (samarpanB) [reply package.json:49]: Yes its a breaking change. So I agree to go ahead with it.

[the remark this answers is not in the document]
Comment (achrinza) [reply package.json:49]: +1 for making a breaking change

[the remark this answers is not in the document]
Comment (samarpanB) [reply package.json:49]: Done

**Comment by samarpanB** [comment]
All checks passing.

**Review by achrinza — APPROVED** [review]
All checks passing; LGTM :+1:

**Pull request merged**